driver/src/main.rs

```diff
@@ -212,8 +212,7 @@ fn main() {
     let price_finder = price_finding::create_price_finder(fee, options.solver_type, price_oracle);
 
     // Create the orderbook reader.
-    let primary_orderbook =
-        PaginatedStableXOrderBookReader::new(contract.clone(), options.auction_data_page_size);
+    let primary_orderbook = EventOrderbook::new(contract.as_ref(), web3);
```

Comment:
I think we want the `Options` struct to have a new option so we can configure at runtime which reader to use, either `OnchainOrderBookReader` shadowed by the `EventOrderbook` or the other way around.

It would be something similar to what we did with the  `scheduler` option and the `SchedulerKind` enum just above.

Reply:
This is part of a different PR #773 unless you mean something different here.

Reply:
Actually yes, this PR is now meant to be the second in a series of two. #773 introduces the configurability while this will simply add the new Event Orderbook to the configuration possibilities.